Sources/SDK/Requests/CartRequest.swift

```diff
@@ -153,7 +153,7 @@ public class CartRequest: MoltinRequest {
      */
     @discardableResult public func removeItem(_ itemID: String,
                                               fromCart cart: String,
-                                              completionHandler: @escaping ObjectRequestHandler<Cart>) -> MoltinRequest {
+                                              completionHandler: @escaping ObjectRequestHandler<CartItem>) -> MoltinRequest {
```

Comment:
This should be an array an cart items - [CartItem]

Reply:
same fix is also needed for sure for `updateItem` at line 180 in CartRequest.swift

and i expect this ones:
`addCustomItem`
`addPromotion`

and maybe this:
`deleteCart`

Instead of `ObjectRequestHandler<Cart>)`
Instead of `ObjectRequestHandler<[CartItem]>)`